lib/split/experiment.rb

```diff
@@ -128,11 +128,13 @@ def alternatives=(alts)
     end
 
     def winner
-      experiment_winner = redis.hget(:experiment_winner, name)
```

Comment:
This one could be dynamic since the winner can be chosen using the UI. If we want to cache this, we probably need a timed eviction of maybe around 5 minutes.

Reply:
Hmmm ... would be pretty straightforward to add a TTL to the cache, but I'm OK letting the experiment run until the next restart ;-)

Reply:
Actually, the cache should just pass through on a `nil` value and keep retrying.  I think this will only cache defined winners and keep polling for experiments that don't have a winner yet.

Reply:
yup....verified that. Only a problem if the winner can be changed, which seems like a fringe use case.

Reply:
The winner is usually set via the dashboard, and does not change usually. Unless the experiment is reopened or something like that.

To keep the cache impl more complete I think it would make sense to also invalidate :experiment_winner on #winner= and #reset_winner:
https://github.com/splitrb/split/blob/master/lib/split/experiment.rb#L158-L161
https://github.com/splitrb/split/blob/master/lib/split/experiment.rb#L144-L148 

It's not bulletproof since It would be a problem if the app instance is deployed on multiple containers. We could end up in a situation where some containers have the correct cache, and some don't. It's not a big deal since this is an 'admin' change. 

With that in mind having a TTL would be nice though. 🙌